rm/rm.c

```diff
@@ -1506,6 +1571,10 @@ M0_INTERNAL int m0_rm_borrow_commit(struct m0_rm_remote_incoming *rem_in)
 		 (long long unsigned) INCOMING_CREDIT(in));
 	M0_PRE(in->rin_type == M0_RIT_BORROW);
 
+	debtor = m0_rm_remote_find(rem_in);
+	M0_ASSERT(debtor != NULL);
+	M0_RM_REMOTE_PUT(debtor);
```

Comment:
Should this be moved downward to LINE 1684? After the debtor is already used and not needed any longer.

Reply:
No, this put() is just to compensate the get() from find(). The needed get() was already taken earlier at request_pre_process(). Let me leave a comment about this...